pages/index.tsx

```diff
@@ -2,7 +2,9 @@ import Head from "next/head";
 import { Map } from "@/components/map";
 import { Header } from "@/components/header";
 import theme, { SourceSans } from "@/utils/theme";
-import { useFetchBikeShare } from '@/hooks/useFetchBikeShare';
+import { useFetchBikeShare } from "@/hooks/useFetchBikeShare";
```

Comment:
ugh I think your ide is doing single quote formatting and mine is double...we should probably add this to the list of things for the pre-commit hook to decide

Reply:
oh no sorry!